NewForce-at-Mountwest/bangazon-site-take-2-fluffy
# Upload a photo when you create a new product

# Description

You can now upload a photo when you create a new product!


## Related Ticket(s)
14 - https://trello.com/c/VOQxi9Oa/14-14-users-can-upload-a-photo-of-the-product


## Steps to Test
--Save, add and commit on your branch before checking out to sw-photos
--in the Package Manager Console, you are going to have to run the command 'update-database' to alter the product table.  If it fails, you might have to nuke the database and start over.
--run the app and navigate to "sell a product"
--add a product to sell and upload an image to go along with it.
--You should see the product details view with the newly included image.
--If you go 'back to list' you should also see a tiny thumbnail of the product on the index view.

# Checklist:
